templates/ja_company/css/virgo.css

```diff
@@ -1567,6 +1567,7 @@ textarea.form-control {
   font-weight: 400;
   font-size: 11px;
   border: 0;
+  margin-bottom: 15px;
```

Comment:
could you send the new views as screenshots on high-width & low-width?

Reply:
**high**
<img width="420" alt="Screen Shot 2019-11-28 at 16 58 47" src="https://user-images.githubusercontent.com/784409/69812028-762a3900-1200-11ea-9cca-7ddb3b958140.png">
**low**
<img width="406" alt="Screen Shot 2019-11-28 at 16 58 58" src="https://user-images.githubusercontent.com/784409/69812034-79252980-1200-11ea-9289-a6d03b947180.png">